subprojects/test-runner/test-inhouse-runner/src/main/kotlin/com/avito/android/rule/ActivityScenarioRule.kt

```diff
@@ -9,15 +9,20 @@ import com.avito.android.rule.base.BaseActivityScenarioRule
  * actions that send out intents which need to be verified or stubbed. This is similar to
  * MockitoAnnotations.initMocks.
  */
-class ActivityScenarioRule<T : Activity>(activityClass: Class<T>) : BaseActivityScenarioRule<T>(
+class ActivityScenarioRule<T : Activity>(
+    activityClass: Class<T>,
+    private val stubIntents: Boolean
```

Comment:
Do we need a default value there?

Reply:
I think no, we don't. I've added `internal` modifier to this class. It's used only in one place.